vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -16197,6 +16257,16 @@ unit:PERCENT-OHM-PER-OHM
   rdfs:label "PERCENT OHM PER OHM" ;
   skos:prefLabel "percent ohm per ohm" ;
 .
+unit:PERCENT-SEC-PER-SEC
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1.0e-00 ;
```

Comment:
Using the same logic as with PERCENT-MI-PER-MI, shouldn't this conversionMultiplier also be 1e-02? Basically just converting a percent into a decimal, right?